sbtplugin/src/main/scala/com/typesafe/tools/mima/plugin/MimaKeys.scala

```diff
@@ -8,7 +8,6 @@ object MimaKeys extends MimaKeys
 class MimaKeys {
 
   final val mimaFailOnProblem      = settingKey[Boolean]("if true, fail the build on binary incompatibility detection.")
-  final val mimaFailOnNoPrevious   = settingKey[Boolean]("if true, fail the build if no previous artifacts are set.")
```

Comment:
Should we deprecate it rather than removing?

Reply:
It's deprecated, but I just moved it down the file (so adding `@deprecated` wouldn't destroy the vertical alignment.)